src/api/index.js

```diff
@@ -1,8 +1,14 @@
+import axios from 'axios';
 // we will define a bunch of API calls here.
 
 const sleep = (time) =>
   new Promise((resolve) => {
     setTimeout(resolve, time);
   });
 
-export { sleep };
+const getExampleData = () => {
+  return axios.get(`https://jsonplaceholder.typicode.com/photos?albumId=1`)
+    .then(response => response.data)
```

Comment:
I'm going to be nit-picky here and ask for semi-colons until we standardize. I think they're good for learners to have to use. Again, our linter/prettier formatter config ought to pick these up. Setting up an issue on that now.